test/openapi3/general/pets-general.yaml

```diff
@@ -83,7 +83,7 @@ paths:
                 $ref: "#/components/schemas/Error"
 servers:
 - url: http://petstore.swagger.io
-- url: http://petstore.swagger.io/staging/
+- url: /staging/
 components:
```

Comment:
Would be better to add additional entry and not replace existing one .

Reply:
i'd also like to see a test that checks that all urls are parsed correctly